### PR TITLE
Suggesting a few ideas

### PR DESCRIPTION
**Pull request by Ivansete-status**
* Secure DeFi
* News over Waku
* Budged approval app
~~* Property registry app~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,10 @@
 - Customer support chat bubbles : A web widget that can be embedded in any website to facilitate messaging between a customer and a customer support agent.
 - Community chat : Chats that are gated to specific communities that own a specific NFT/Token/Onchain proof.
 - Authentication methods : New methods to authenticate a user while using a dapp. For example : connect your wallet by scanning a QR code that holds a URI on Waku.
+- Secure DeFi: When a user wants to perform a transaction, ensure the transaction is made to the correct counter-party address within the correct chain.
+- News over Waku: Platform where people can publish news which can be voted and commented.
+- Budget approval app: Allow a community to approve and decide the best way to handle the common funds, and make the transactions transparent to the community.
+- Property resitry app: Real-state (buildings, flats, lands) registry where buy/sell operations are tracked.
```
Comment (vpavlin) [inline README.md]: s/resistry/registry/

**Comment by vpavlin** [comment]
Can you elaborate on where Waku fits into these? I may have some thoughs, but would like to get your view since they are ideas:)

**Comment by Ivansete-status** [comment]
> Can you elaborate on where Waku fits into these? I may have some thoughs, but would like to get your view since they are ideas:)

Sure! Let me elaborate on them a bit more:

- **Secure DeFi**
In blockchain transactions, there is a high risk of mistyping an address or picking up an incorrect on-chain network. That leads to an unrecoverable loss to the user if something is not configured properly.

  This app is aimed to add an additional security layer where the payer can ensure that s/he is setting the correct parameters and that their funds will be properly transmitted.
  
  Waku can help with this "transaction confirmation" protocol.

- **News over Waku**
  This app will allow users to perform the following actions:
   - Publish news.
   - Vote news up or down.
   - Add comments to news.
   - Vote comments to news up or down.
   
  This app matches perfectly with Waku as it will represent a source of information without censorship.
  We can encourage the developer to use Codex as a persistence system.

- **Budget approval app**
  This app will help people to reach an agreement on any particular expenses. This app could be a combination of chat plus a voting mechanism where agreements are met.

- **Property registry app**
  As @vpavlin already commented to me on Discord, this would have more on-chain development so maybe is not a good candidate to explore the Waku capabilities.

**Comment by jm-clius** [comment]
Great to see. "News over Waku" and "Budget approval" especially make sense to me in terms of Waku usage. Not quite clear on "Secure DeFi" over Waku as this sounds like quite a big undertaking. Do you mean to establish secure chat channels between the target and destination Wallets using Waku? Or some other confirmation handshake? Seems like an interesting idea, though the actual confirmation protocol may be the bulk of the work with Waku being a fairly straightforward transport (not that that is a problem).

**Comment by Ivansete-status** [comment]
> Do you mean to establish secure chat channels between the target and destination Wallets using Waku? Or some other confirmation handshake? Seems like an interesting idea, though the actual confirmation protocol may be the bulk of the work with Waku being a fairly straightforward transport (not that that is a problem).

Yes, I mean "to establish a secure chat channel between the target and destination Wallets using Waku". That "confirmation protocol" can be developed on the top of Waku.

**Comment by gabrielmer** [comment]
Really liked the idea of "News over Waku" also! I think it is in a way similar to "Decentralized Social Media" that many people are trying to solve, but much more straightforward and especially useful.

However, same as social media content, I think there's a component of being particularly important for news to be securely archived forever. In those cases there's generally some on-chain component, please correct me if I'm wrong.

There's an use case of "real time, p2p short news/alerts" in which persistence is less important of a feature. Those news are generally shared between people on WhatsApp or Telegram, and Waku is an amazing use case for that in my opinion.

Anything related to information sharing where censorship is an issue is a top use case for Waku. Will also think about it more and share here :)

**Comment by hackyguru** [comment]
1. exchanges have been doing a centralised version of verifying a transaction and waku can be really useful in this aspect

2. +1 for news over waku, it's a really easy to implement but important use case...

3. a decentralized version of expensify :) wow!

4. property registration : i'd say saving information related to real world assets on waku might not be feasible because waku can not be compared to on-chain storage. maybe this should be a use case for nomos not waku

**Comment by gabrielmer** [comment]
Another application in which I think that Waku could be a great fit, which is also related to the news idea, is a decentralized fact checking protocol. Like Twitter/X's Community Notes, which if I'm not mistaken is designed/intended to be decentralized, but can't be while it's run on a centralized entity's servers.

**Review by hackyguru — APPROVED** [review]

**Comment by hackyguru** [comment]
closing this PR bevcause the the idea board is now in a different layout. i'll manually add these ideas

**Pull request closed without merging**